docs/conf.py

```diff
@@ -23,9 +23,11 @@
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
 
+ROOT_PATH = os.path.dirname(os.path.realpath(__file__))
+
+sys.path.insert(0, os.path.join(ROOT_PATH, 'ext'))
```

Comment:
Before this there was `ROOT_PATH + '/utils'`, but now there's `/ext` -- what is the difference?

Reply:
`/utils` is unrelated to `/ext`. It was here before, apparently so that you can import stuff from a `utils` subdirectory. This directory does not exist and does not seem like we're not using any utils so I removed it.

Also, it was inside `setup()` and that's too late. `/ext` must be added at module level for Sphinx to be able to load extensions from there.